src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -26,7 +27,7 @@ beforeEach(() => {
   jest.spyOn(Router, 'useParams').mockReturnValue({
     id: '123',
   });
-  jest.spyOn(Router, 'useHistory').mockReturnValue({});
+  jest.spyOn(Router, 'useHistory').mockReturnValue({} as any);
```

Comment:
Maybe could use render from testUtils to get rid of router mocks and apollo mocked provider